I [made the change](https://github.com/wix/react-fastclick/commit/5dbc69d8c1ebdf8135ec06b5eb771bf80e5a8afc) you asked for (I hope it's ok, I'm not too familiar with AMD/CommonJS import mechanisms).

But I ran into the following eslint error which I couldn't solve:

```sh
/Users/dutzi/Documents/wix/react-fastclick-fork/lib/index.js
  247:33  error  'React' is already declared in the upper scope  no-shadow
```

--- a/lib/index.js
+++ b/lib/index.js
@@ -244,8 +244,8 @@
     module.exports = getReactFCInitializer(React);
   // Export for amd / require
   } else if (typeof define === 'function' && define.amd) { // eslint-disable-line no-undef
-    define(['react'], function (React) { // eslint-disable-line no-undef
-      return getReactFCInitializer(React);
+    define(['react'], function (ReactAMD) { // eslint-disable-line no-undef
+      return getReactFCInitializer(ReactAMD);
     });
   // Export globally
   } else {

